queryfilter/datetimefilter.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import absolute_import
+
+import datetime
+
+from dateutil import parser
+import pytz
+
+from .base import FieldFilter, DictFilterMixin, DjangoQueryFilterMixin
+from .queryfilter import QueryFilter
+
+
+@QueryFilter.register_type_condition('datetime')
+class DatetimeRangeFilter(DjangoQueryFilterMixin, DictFilterMixin,
+                          FieldFilter):
+
+    @property
+    def start(self):
+        return get_start(self.filter_args.get("start"))
+
+    @property
+    def end(self):
+        return get_end(self.filter_args.get("end"))
+
+    def on_dicts(self, dicts):
+
+        def in_range(datum):
+            datetime_string = self.get(datum, self.field_name)
+            if isinstance(datetime_string, datetime.datetime):
+                to_compare = datetime_string
+            else:
+                to_compare = parse(datetime_string)
+
+            if self.start and (to_compare < self.start):
+                return False
+
+            if self.end and (self.end < to_compare):
+                return False
+
+            return True
+
+        return list(filter(in_range, dicts))
+
+    def do_filter(self, queryset):
```

Comment:
`do_filter` looks like dedicated to `Django` query. What if one day it also needs to support `SQLAlchemy` or `PonyORM`? Since `on_dict` is here too, maybe `on_django_query`?